src/common/Button.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { StyledButton } from './CommonHTML'
+
+export default function Button(props) {
+  const { buttonText, clickEvent } = props
+  return (
+    <StyledButton onClick={clickEvent}>{buttonText}</StyledButton>
+  )
+}
```

Comment:
You are doing great. It would be a better practice if you:
1. create a folder named `Button`
2. create a file named `index.js`
3. place both `button.js` and `index.js` inside the Button folder
```js
// index.js
import Button from './Button';
export default Button;
```

This will give us some benefits:

- Easy update `Button` but not too much update in the `Button.jsx`
- Easy to allocate the target component file, if you use `index.js` for all the components, it would be difficult to figure out.
- Keep the same way to import and export components.


It will look like this after change:
```bash
- Button
  - index.ts
  - Button.jsx
```

This could be applied to all other components. Thanks.